pkg/registry/policy/poddisruptionbudget/strategy.go

```diff
@@ -91,9 +91,10 @@ func (podDisruptionBudgetStrategy) ValidateUpdate(ctx api.Context, obj, old runt
 	return append(validationErrorList, updateErrorList...)
 }
 
-// AllowUnconditionalUpdate is the default update policy for PodDisruptionBudget objects.
+// AllowUnconditionalUpdate is the default update policy for PodDisruptionBudget objects. Status update should
```

Comment:
AllowUnconditionalUpdate is used only if the resource version in the UPDATE request is zero:
https://github.com/kubernetes/kubernetes/blob/c41c24fbf300cd7ba504ea1ac2e052c4a1bbed33/pkg/registry/generic/registry/store.go#L377
How is this helping?